### PR TITLE
- Added support for Unity's ISerializationCallbackReceiver interface

### PR DESCRIPTION
**Pull request by lazlo-bonin**

### CODE REVIEW

File: Source/fsISerializationCallbacks.cs [hunk 1]
```diff
@@ -60,4 +60,22 @@ public class fsSerializationCallbackProcessor : fsObjectProcessor {
             ((fsISerializationCallbacks)instance).OnAfterDeserialize(storageType);
         }
     }
+
+#if !NO_UNITY
+
```
Comment (jacobdufault) [inline Source/fsISerializationCallbacks.cs]: Can you remove this extra blank line?

File: Source/fsSerializer.cs [hunk 2]
```diff
@@ -283,7 +283,11 @@ internal class fsLazyCycleDefinitionWriter {
                 new fsSerializationCallbackProcessor()
             };
 
-            Context = new fsContext();
+#if !NO_UNITY
```
Comment (jacobdufault) [inline Source/fsSerializer.cs]: Please use spaces here.

**Comment by jacobdufault** [comment]
Looks good, would you mind making the commented style fixes? I'll merge after that.

**Comment by lazlo-bonin** [comment]
Finally got to it!

**Comment by jacobdufault** [comment]
Thanks!

**Pull request merged**